fastlane/metadata/ratings_config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "alcoholTobaccoOrDrugUseOrReferences": "NONE",
+  "contests": "NONE",
+  "gamblingSimulated": "NONE",
+  "horrorOrFearThemes": "NONE",
+  "matureOrSuggestiveThemes": "NONE",
+  "medicalOrTreatmentInformation": "NONE",
+  "profanityOrCrudeHumor": "NONE",
+  "sexualContentGraphicAndNudity": "NONE",
+  "sexualContentOrNudity": "NONE",
+  "violenceCartoonOrFantasy": "NONE",
+  "violenceRealisticProlongedGraphicOrSadistic": "NONE",
+  "violenceRealistic": "NONE",
+  "gambling": false,  
+  "seventeenPlus": false,
```

Comment:
Shouldn't this be set to `"seventeenPlus": true` – and shouldn't the `"matureOrSuggestiveThemes"` be set to "some" or whatever – to address paaHJt-2bZ-p2 ?

Reply:
`matureOrSuggestiveThemes` in this case, no – the `unrestrictedWebAccess` is the one that best fits our profile, and causes the app to be bumped to `17+`

Reply:
Strange that there's a `seventeenPlus: false` in the JSON if the age rating is determined by the other factors (and I don't think we can select 17+ true/false in ASC's UI anyway, only the other non-boolean categories listed in the json). Is that a legacy field? (if so, couldn't it be completely omitted from the JSON to avoid confusion?)